src/purity.erl

```diff
@@ -1254,7 +1254,7 @@ converge_pst(Fun, #pst{table = Tab} = St0) ->
             converge_pst(Fun, St1)
     end.
 
-%% @doc Return two lists of functions with concrete values, one for 
```

Comment:
Unintentional white space change. Sorry about that (my editor removes this kind of occurrence, but I usually don't commit it, so this was oversight, here).